code/API_definitions/check-device-connectivity.yaml

```diff
@@ -158,6 +162,16 @@ components:
       enum:
         - ROAMING_ON
         - ROAMING_OFF
+    CountryCode:
+      description: The Mobile country code (MCC) as an geographic region identifier for the country and the dependent areas.
+      type: integer
+      example: 340
+    CountryName:
+      description: A list of ISO 3166 ALPHA-2 country-codes of mapped to the mobile country code (MCC).
+      type: array
+      items:
+        type: string
+      example: [BL,GF,GP,MF,MQ]
```

Comment:
I'd like to see multiple examples here (using keyword `examples` rather than `example`):
- No associated country `[]`
- Single associated country e.g. `["DE"]`
- Multiple associated countries e.g. `["BL","GF","GP","MP","MQ"]`

The current example with multiple countries would be rare in practice. The usual response would be a single country, so the description should be extended to explain why multiple countries or no country may be returned.

My current preference for no associated country is an empty array, but an alternative would be `["XX"]`.

Reply:
![Multiple Country Code](https://user-images.githubusercontent.com/117813163/218800896-59a232fc-adcf-4caa-a787-2ee826664664.PNG)
![Single Country Code](https://user-images.githubusercontent.com/117813163/218800910-7a87115a-c8e0-4b51-aea7-65130c3fe0d2.PNG)
![No Country Code](https://user-images.githubusercontent.com/117813163/218800920-4af56264-7002-41e7-b652-11e18f089473.PNG)

Reply:
We Updated YAML with all three Examples with extended description.

Reply:
Hello @ravindrapalaskar17 
I agree with @eric-murray about "No associated country" case. I think an empty array [] is a better choice. Is there any reason why you prefer to return ["XX"]?

Reply:
@ravindrapalaskar17 let's go with an empty array for an undefined countryName.

Reply:
Thanks for feedback we updated YAML with empty array.